.github/workflows/seed-data.yml

```diff
@@ -46,8 +44,8 @@ jobs:
         if: ${{ inputs.core-image-tag }}
         run: |
           cd opencrvs-core
-          git checkout ${{ inputs.core-image-tag }}
```

Comment:
Let's keep using `inputs` as it works for both workflow dispatch & reusing workflows